app/server/lib/OIDCConfig.ts

```diff
@@ -35,6 +35,16 @@
  *    env GRIST_OIDC_SP_IGNORE_EMAIL_VERIFIED
  *        If set to "true", the user will be allowed to login even if the email is not verified by the IDP.
  *        Defaults to false.
+ *    env GRIST_OIDC_IDP_ENABLED_PROTECTIONS
+ *        A comma-separated list of protections to enable. Supported values are "PKCE", "STATE", "NONCE".
+ *        It's highly recommended that you enable STATE, and at least either PKCE or NONCE.
+ *        Defaults to "PKCE,STATE".
+ *    env GRIST_OIDC_IDP_ACR_VALUES
+ *        A space-separated list of ACR values to request from the IdP. Optional.
```

Comment:
specific reason to do coma separated values and after space-separated list ?

Reply:
Yep, here it is because of the spec:
https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest

> OPTIONAL. Requested Authentication Context Class Reference values. **Space-separated string** that specifies the acr values that the Authorization Server is being requested to use for processing this Authentication Request, with the values appearing in order of preference. The Authentication Context Class satisfied by the authentication performed is returned as the acr Claim Value, as specified in [Section 2](https://openid.net/specs/openid-connect-core-1_0.html#IDToken). The acr Claim is requested as a Voluntary Claim by this parameter.

Reply:
Does it answer your question?